e2e/pages/adf/dataTablePage.ts

```diff
@@ -35,6 +35,7 @@ export class DataTablePage {
     reset = element(by.xpath(`//span[contains(text(),'Reset to default')]/..`));
```

Comment:
This file still has a lot of duplicating and unnecessary methods. What i identified until now: 
- clickMultiselect(), clickReset(), selectSelectionMode(), addRow(),replaceRows(), replaceColumns(), insertFilter() is part of demo-shell. Please create a datatablePage in demo-shell folder and move these methods in there;
- selectRowWithKeyboard(): it takes the row by number and we don't want to take it for specific column.
- getNrOfRows - duplicates getAllDisplayedRows. Please keep only one.
- getNodeIdFirstElement() : it is specific to a column, please delete it and use a general method that takes 'Node Id' from a demo-shell page.
- please remove selectRowByContentName(). It duplicates selectRow()
- please remove checkRowIsDisplayedByName and checkRowIsNotDisplayedByName. These duplicates checkContentIsDisplayed and checkContentIsNotDisplayed